Controller/ContentTypeController.php

```diff
@@ -130,7 +130,7 @@ public function createContentTypeAction($contentTypeGroupId, $languageCode = nul
             $contentTypeGroup = $this->contentTypeService->loadContentTypeGroup($contentTypeGroupId);
 
             $contentTypeCreateStruct = new ContentTypeCreateStruct([
-                'identifier' => 'new_content_type',
+                'identifier' => '__new__' . md5(microtime(true)),
```

Comment:
Yeah, this is not meant to appear in UI. This identifier is purely internal, to ensure that it's unique.